docs/smart-contracts/anatomy/yield-resume.md

```diff
@@ -197,6 +197,16 @@ The reason to not raise an error, is because we are changing the state (removing
 
 ---
 
+## Managing State
+
+When using yield and resume, it's important that you carefully manage the contract's state. Because of its asynchronous execution, the yield and the resume happen, and are finalized in separate blocks.
```

Comment:
```suggestion
When using yield and resume, it's important that you carefully manage the contract's state. Because of its asynchronous execution, the contract function in which the contract yields and resumes are independent.
```